README.md

```diff
@@ -1,2 +1,228 @@
-# example-application
-Example out-of-tree application that is also a module
+![Zephyr Project logo](https://github.com/zephyrproject-rtos/zephyr/blob/v1.14-branch/doc/images/Zephyr-Project.png)
+
+
+The Zephyr Project is a scalable real-time operating system (RTOS) supporting
+multiple hardware architectures, optimized for resource constrained devices,
+and built with security in mind.
+
+
+# Getting Started
+
+The Zephyr build system supports code tree "plug-ins" called *modules*.
+See the [Zephyr Modules (External projects)][3] for a high-level overview
+of the build system module mechanism.
+
+The Zephyr eXample Module (ZXA) is supplementary to the Zephyr repository.
+This provides:
+
+1. A reference implementation of using the Zephyr module mechanism.
+
+1. A repository for holding example code and config files outside of the
+   Zephyr repository, including:
+
+   - `west` manifest
+   - projects/samples/tests
+   - applications/subsystems/drivers/interface declarations
+   - device tree config files
+   - boards
+   - scripts
+
+1. A test case for CI to monitor Zephyr build system support for modules.
+
+
+   _WARNING:_ The Zephyr repository is intentionally
+   **independent of and unaware of this repository**.
+
+
+The ZXA module is designed around the following principles:
+
+- Keep all Zephyr-specific content within the (required) `zephyr` folder.
+
+  - This enables adding the example on to an existing repository.
+
+- Follow the Zephyr repository structure, mechanisms, and rules
+  whenever possible.
+
+  - This minimizes the variance from what users and developers expect,
+    simplifies maintenance, and reuses existing solutions.
+
+- Extend the Zephyr repository structure to support an arbitrary number
+  of applications and project configurations.
+
+  - The `projects` sub-tree identifies where non-test, non-sample projects
+    are expected to be found.  The `projects` folder is tied into the
+    Zephyr CI system through `module.yml`.
+
+- Maximize the amount of code automatically verifiable through Zephyr CI.
+
+- Minimize risk of conflicts in global namespaces through use of the
+  module abbreviation.
+
+  - Global namespaces exist in many areas, including:
+    Kconfig, CMake, C preprocessor symbols; include file pathnames;
+    test and sample names; shell commands; settings identifiers.
+    Using the module abbreviation as a global symbol prefix reduces
+    the chance of collisions as other modules are created.
+
+## Example vs Sample
```

Comment:
Since we are calling this an "example" rather than a "sample", I think we really need this clarified *somewhere*. It has been relocated to the bottom of the landing page. Once we figure out what needs to happen with module releases & release notes/documentation, we can consider splitting content into multiple pages.